docker/ssl/nginx-image/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:stable
+
+# Install dependencies and then remove apt-get lists to save space
+RUN apt-get update && apt-get install -y procps && rm /var/lib/apt/lists/*
```

Comment:
`rm /var/lib/apt/lists/*` you need to make this command recursive `rm -rf rm /var/lib/apt/lists/*`

Reply:
🤦‍♂️ not sure how I missed that